### ISSUE TITLE
perspective image branch

### ISSUE THREAD

**Issue by quinnzjh** [Insta360-Research-Team/DiT360#14]
congratulations to your rencently works！
 i cant find the perspective image repojective branch called by your paper  from your codes.could you interprete it again? 

`        model_pred = self.flux_transformer(
            hidden_states=packed_noisy_model_input,
            timestep=timesteps / 1000,
            guidance=guidance_vec,
            pooled_projections=pooled_prompt_embeds,
            encoder_hidden_states=prompt_embeds,
            txt_ids=text_ids,
            img_ids=latent_image_ids,
            return_dict=False,
        )[0]
`
loss:
`
            loss = loss_v + self.hparams.args.lambda_yaw * loss_yaw + self.hparams.args.lambda_cube * loss_cube

`
